docs/source/basic/terms.rst

```diff
@@ -0,0 +1,57 @@
+Terms
+=====
+
+This page provides an overview of the terms used in ``alpaka`` and the relationships between them.
+
+.. image:: /images/alpaka_terms.svg
+
+Platform
+--------
+
+- A ``platform`` contains information about the system, e.g. the available devices. 
+- Depending on the platform, it also contains a runtime context.
+- A ``platform`` has a least one device but it can also has many device.
+- Each ``platform`` can be used with N ``accelerator``. ``platforms`` and ``accelerator`` cannot freely combined. An ``accelerator`` supports only a specific ``platform``.
+ 
+Device
+------
+
+- A ``device`` represent a compute unit, such as a CPU or a GPU.
+- Each ``device`` is bounded to a specific ``platform``.
+- Each ``device`` can have N ``queues``.
+
+Accelerator
+-----------
+
+- A ``accelerator`` is a index mapping function. It distributes the index space to the chunks. The ``accelerator`` maps a continues index space to a blocked index domain decomposition.
```

Comment:
It is much more than that.
For example, the OpenMP and TBB accelerators implement different kind of parallelism for the Cpu device.

Reply:
Yes, I know. Accelerator is pretty hard to explain. Actual I only wrote down, what @psychocoderHPC told me, because my personal description is to inprecisely.

Reply:
I would discuss the documentation of accelerator in the next alpaka VC. For me, this is a complicated subject.